### PR TITLE
fix: adapt journal for cmd

### PR DESCRIPTION
**Pull request by jingjunLi**
### Description

A new JournalFilePath configuration has been added to triedb.Config, fix some code to adapt this config to cmd;

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -2430,6 +2430,12 @@ func MakeBlockDatabase(ctx *cli.Context, stack *node.Node, readonly, disableFree
 	return blockDb
 }
 
+func PathDBConfigAddJournalFilePath(stack *node.Node, config *pathdb.Config) *pathdb.Config {
+	path := stack.ResolvePath("chaindata") + "/" + eth.JournalFileName
```
Comment (sysvm) [inline cmd/utils/flags.go:2434]: Use `fmt.Sprintf()` is better.

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -2593,6 +2599,7 @@ func MakeTrieDatabase(ctx *cli.Context, disk ethdb.Database, preimage bool, read
 	} else {
 		config.PathDB = pathdb.Defaults
 	}
+	config.PathDB.JournalFilePath = stack.ResolvePath("chaindata") + "/" + eth.JournalFileName
```
Comment (sysvm) [inline cmd/utils/flags.go:2602]: ditto

**Review by sysvm — APPROVED** [review]
LGTM

**Review by RenRick — APPROVED** [review]
LGTM

**Pull request merged**